src/components/Popover/Popover.module.css

```diff
@@ -0,0 +1,137 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # POPOVER
+\*------------------------------------*/
+
+/**
+ * 1) Box that contains content
+ */
+.popover {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-start;
+  visibility: hidden;
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: var(--eds-z-index-200) 0;
+  background-color: transparent;
+  transition: all var(--eds-anim-fade-long) var(--eds-anim-ease);
+  transform: translateZ(0);
+
+  &.eds-is-active {
+    visibility: visible;
+  }
+
+  &:focus-visible {
+    @mixin focus;
+  }
+}
+
+/**
+ * Popover window
+ * 1) Actual box with content within
+ */
+.popover__window {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: auto;
+  left: 0;
+  bottom: 68px;
+  height: auto;
+  max-height: 450px;
+  width: 400px;
+  max-width: 400px;
+  margin-left: 13px;
+  opacity: 0;
+  background: var(--eds-color-neutral-white);
+  border-radius: var(--eds-theme-border-radius);
+  box-shadow: var(--eds-theme-box-shadow);
+  transition: opacity var(--eds-anim-fade-quick) var(--eds-anim-ease);
+  overscroll-behavior-y: contain;
+
+  &:focus-visible {
+    @mixin focus;
+  }
+
+  /**
+  * Popover arrow
+  * 1) A 0 height/0 width empty container positioned at the bottom of the popover; its purpose is to provide...
+  * 2) ...a psuedo element that adds a triangle added to the bottom edge of the popover pointing toward the button that opened the popover
+  */
+  .popover__arrow {
+    position: relative;
+    bottom: 0;
+    left: 0;
+  }
+  &::after {
```

Comment:
Can we add this as a regular CSS comment above the &::after. With &:hover and stuff comments aren't necessary, but for these I like to comment them since they are more obscure. Thanks!

Reply:
Moved to a new comment